src/providers/data.rs

```diff
@@ -294,19 +293,13 @@ pub trait Format: Sized {
 
     /// Parses `string` as the data format `Self` as a `T` or returns an error
     /// if the `string` is an invalid `T`.
-    fn from_str<'de, T: DeserializeOwned>(string: &'de str) -> Result<T, Self::Error>;
+    fn from_str<T: DeserializeOwned>(string: &str) -> Result<T, Self::Error>;
```

Comment:
I don't think this is backwards compatible.

Let's remove this change and just leave a note in the source code about changing it in the future.

Reply:
Ah, because this is a trait that might be implemented by other crates. Got it.